features/yield/FarmingCalculator.tsx

```diff
@@ -186,6 +194,8 @@ const FarmingCalculator = () => {
     }
     if (Object.keys(WEEKLY_UMA_REWARDS).includes(tokenAddress)) {
       setRewardToken(WEEKLY_UMA_REWARDS[tokenAddress]);
+    } else {
+      setRewardToken(null);
```

Comment:
This hides the farming calc if there is no WEEKLY_UMA_REWARDS info for the token address